### DISCUSSION TITLE
Hospedagem do Backend e Front End

### DISCUSSION THREAD

**Discussion by fe-lipejs** [frontendbr/forum#2314] [Perguntas e Respostas]
Olá a todos, sou novo na área. 
Eu tenho aplicação front que está separada da backend.
Estou querendo hospedar, o meu site(back e front). Qual a melhor forma de fazer? Será que existe uma forma de hospedar o front e o back com o mesmo endereço, ou vou ter que criar dominios diferentes? 
Espero ter sido claro, obrigado a todos!

**Comment by felquis** [comment]
O que eu ando mais fazendo é assim:

meusite.com.br hospedado na vercel.
api.meusite.com.br hospedado na digital ocean.

**Reply by jeeanribeiro** [reply]
É um bom combo, plataformas que facilitam na configuração

**Comment by BrunoMCampos** [comment]
Oi, pessoal, tudo bem? Estou iniciando na área agora também e estou com uma dúvida, poderiam me ajudar?
Também tenho o front e o back prontos de uma aplicação web, backend feito em Java e front em React, como posso fazer o deploy das duas?
Na vercel eu já tenho algumas aplicações upadas, é mais tranquilo, meu problema está sendo com o backend, na realidade.
Por exemplo, na Digital Ocean, como funcionaria para hospedar minha API? Qual plano teria que utilizar? Teria apenas que realizar o upload da build e rodar como se estivesse rodando em uma virtual machine? Como funciona a parte de criação do banco de dados e tudo mais?
Realmente não tenho muito conhecimento de quase nada, como podem ver kkkkkk 
Valeu pela força desde já!

**Reply by JoaoVictor6** [reply]
Básicamente vc precisaria de alguma forma rodar o codigo do java. Como eh uma api este codigo tem uma porta como ponti de acesso. Pra fazer isso rodar e ser utilizado vc precisa de uma VPS, acredito.
